common-auth/SECURITY.md

```diff
@@ -0,0 +1,65 @@
+# Running registry and streamline web-services securely
+
+This module is intended to be used by registry and streamline web-services so that they can enable http client authentication via SPNEGO. Currently supported 
+authentication mechanism is Kerberos. The code for this module has been borrowed from hadoop-auth(2.7.3) module in hadoop project and changed a little. The 
+reasons for doing so are to avoid having a dependency on hadoop-auth module which brings in some other modules, avoid conflicts with other versions of 
+hadoop-auth module and having more control over the changes needed in future. Some text for this document has been borrowed from SECURITY.md of Apache Storm  
```

Comment:
I don't mind removing that but it loses a lot of information that was suggested by others to be included in this doc. Lets keep it if you don't mind